### ISSUE TITLE
"An Unholy Alliance" Quest (Part 1) Bugged, Uncompletable

### ISSUE THREAD

**Issue by NateNeumann** [dalaranwow/dalaran-wow#5163]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Bringing Small Scroll to the Bragor Bloodfist does not give the option to complete the quest, and no question mark appears above his head.

**Expected behaviour**: Tell us what should happen instead
Should be able to turn in quest.

**Steps to reproduce the problem**:

1. I think it's probably universal, as others have reported the same bug on other forums. But I'm a Tauren Druid, if that helps anything,..
2.  Receive Small Scroll (in a dungeon, for me)
3. Attempt to turn in quest
4. Unable to.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @NateNeumann,
Issue confirmed on PTR.
![image](https://user-images.githubusercontent.com/6797611/33994345-df93952c-e0e2-11e7-8d4f-03816b1ceaeb.png)

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the report. This issue will be fixed with the next maintenance.

**Comment by NateNeumann** [comment]
Great, thank you!
